packages/address-service-client/index.js

```diff
@@ -0,0 +1,26 @@
+const conf = require('@condo/config')
```

Comment:
This package is used only inside the plugin. It's better to move all this logic inside the plugin. And not to create a new package.

Reply:
Done. Address service client was moved into plugin.